tests/load-tests/ci-scripts/utility_scripts/playwright-update-tokens.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+# Docs:
+#     This script uses credentials (username and password) from users.json
+#     to login to console.dev.redhat.com and generate new offline token. It
+#     saves updated content to users-new.json.
+#
+# Setup:
+#     python -m venv venv
+#     source venv/bin/activate
+#     pip install playwright
+#     playwright install   # download Chromium browser
+#
+# Running:
+#     Consider PLAYWRIGHT_HEADLESS = True so it runs in background
+#
+#     ci-scripts/utility_scripts/playwright-update-tokens.py
+#
+#     To sort final file I have used:
+#     cat users-new.json | jq '. | sort_by(.username[11:] | tonumber)'
+#
+#     Note I was not able to run with more threads as I was getting
+#     "Access Denied" errors. I guess it was some rate limiting.
+
+import playwright.sync_api
+import os
+import time
+import json
+import concurrent.futures
+
+PLAYWRIGHT_HEADLESS = False
+PLAYWRIGHT_VIDEO_DIR = "videos/"
+
+def thread(user):
+    username = user["username"].replace("-", "_")
+    password = user["password"]
+
+    with playwright.sync_api.sync_playwright() as p:
+        browser = p.chromium.launch(
+            headless=PLAYWRIGHT_HEADLESS,
+        )
+        context = browser.new_context(
+            record_video_dir=PLAYWRIGHT_VIDEO_DIR,
+        )
+        page = context.new_page()
+
+        page.goto("https://console.dev.redhat.com")
+        page.wait_for_url("https://sso.redhat.com/**")
+
+        # Accept cookies
+        cookies_iframe = page.frame_locator('iframe[name="trustarc_cm"]')
+        cookies_button = cookies_iframe.get_by_role("button", name="Agree and proceed with standard settings")
+        cookies_button.click()
+
+        # Wait for login form and use it
+        page.wait_for_selector('//h1[text()="Log in to your Red Hat account"]')
+        input_user = page.locator('//input[@id="username-verification"]')
+        time.sleep(1)
```

Comment:
is this necessary?

Reply:
Yep. I spend quite some time trying to figure this out. There is some elements shuffling in the DOM that is happening here or maybe it is some part of some animation, anyway for some reason, this moved success rate of the script from "0 of 10" to "100 of 100" :-(

Reply:
Then this needs to be here :-D, thanks for the info!